search/geocoder.hpp

```diff
@@ -233,7 +232,7 @@ class Geocoder
 
   // This field is used to map features to a limited number of search
   // classes.
-  SearchModel const & m_model;
+  SearchModel m_model;
```

Comment:
No, everything is fine. SearchModel is stateful now, so Geocoder should have its own instance of SearchModel.